### PR TITLE
Add sentence for doing your own research

### PR DESCRIPTION
**Pull request by abolinsky**
[Reference to english file](https://github.com/ms1-learner/cpp-01-en/tree/main/03_organization_and_testing/02_challenge)

### CODE REVIEW

**Comment by abolinsky** [comment]
https://woven-dojo.atlassian.net/browse/CLD-554?atlOrigin=eyJpIjoiMjY1ZjVjZjYzZGZlNDQ1NjllODYwZTYyN2M0Y2UwNWIiLCJwIjoiaiJ9

File: 03_organization_and_testing/02_challenge/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # チャレンジ2
 
-C++11以降では4番目のタイプの記憶域期間が追加されています。何という記憶域期間ですか。この記憶域期間を持つ `int` 型の変数 `thread_data` を宣言し、値10で初期化するには、どのようにしますか。
+C++11以降では4番目のタイプの記憶域期間が追加されています。何という記憶域期間ですか。自分自身で調査する必要があります。この記憶域期間を持つ `int` 型の変数 `thread_data` を宣言し、値10で初期化するには、どのようにしますか。
```
Comment (dancer-yuma) [inline 03_organization_and_testing/02_challenge/README.md:3]:
```suggestion
C++11以降では4番目のタイプの記憶域期間が追加されています。何という記憶域期間ですか。自分で調べてみましょう。この記憶域期間を持つ `int` 型の変数 `thread_data` を宣言し、値10で初期化するには、どのようにしますか。
```

**Review by dancer-yuma — CHANGES_REQUESTED** [review]
@abolinsky 
It is not wrong, but I changed it to be more like a suggestion.

File: 03_organization_and_testing/02_challenge/README.md (hunk 1 above)
Comment (abolinsky) [reply 03_organization_and_testing/02_challenge/README.md]: @dancer-yuma accepted

**Pull request merged**